openreview/api/client.py

```diff
@@ -3326,17 +3334,26 @@ def to_json(self):
         if self.invitation:
             body['invitation'] = self.invitation
 
-        if self.replyto:
-            body['replyto'] = self.replyto
-
         if self.readers:
             body['readers'] = self.readers
 
         if self.nonreaders:
             body['nonreaders'] = self.nonreaders
 
-        if self.signatures:
-            body['signatures'] = self.signatures
+        if self.signature:
+            body['signature'] = self.signature
+
+        if self.profile:
+            body['profile'] = self.profile
+
+        if self.weight:
```

Comment:
According to Sherry, this fails when self.weight == 0. This should fix it:
```suggestion
        if self.weight is not None:
```